app/api/views/briefs.py

```diff
@@ -145,6 +146,167 @@ def get_brief_responses(brief_id):
     return jsonify(brief=brief.serialize(with_users=False), briefResponses=brief_responses)
 
 
+@api.route('/brief/<int:brief_id>/sellers', methods=['GET'])
+@login_required
+@role_required('buyer')
+def get_brief_responded_sellers(brief_id):
+    """All sellers who responded to a now closed brief (role=buyer)
+    ---
+    tags:
+      - "Brief"
+    security:
+      - basicAuth: []
+    parameters:
+      - name: brief_id
+        in: path
+        type: number
+        required: true
+    definitions:
+      BriefResponses:
+        properties:
+          sellers:
+            type: array
+            items:
+              id: BriefResponse
+    responses:
+      200:
+        description: A list of sellers as brief responses
+        schema:
+          id: BriefResponses
+      403:
+        description: brief_id is not owned by the requesting user
+      404:
+        description: brief_id not found
+    """
+    brief = briefs.get(brief_id)
+    if not brief:
+        not_found("Invalid brief id '{}'".format(brief_id))
+
+    if hasattr(current_user, 'role') and current_user.role == 'buyer':
```

Comment:
Interestingly `hasattr()` is not used that much in this code base.  More often `get()` is the the preferred pattern, but  its a philosophical argument.